api/nodes/serializers.py

```diff
@@ -207,6 +208,9 @@ def get_license_details(node, validated_data):
         raise exceptions.ValidationError(detail='License ID must be provided for a Node License.')
     license_id = license.node_license.license_id if license else None
     license_year = license.year if license else None
+    if license_year is None and isinstance(node, DraftRegistration):
+        license_year = str(datetime.datetime.now().year)
```

Comment:
I'm not sure how helpful this change will be; it looks like we are auto-populating the year in the front-end as well. We definitely need to fix L207-208, and ideally we'd make it so that license fields are not required for draft registrations, only for full registrations.